11ty/definitions/nl_NL/cisgender.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Cisgender
+slug: cisgender
+speech: adj
+defined: true
+---
+
+waar de geslachtsidentiteit/genderidentiteit overeenkomt met het geslacht waar men mee geboren is.
```

Comment:
You use geslacht in place of "sekse" here but earlier on you used sekse. Is it better to be consistent and use one over the other throughout the Dutch translation? Or did you intend a semantic difference?

Reply:
Yeah, sekse is not automatically the same as the geslacht you are born with, and that distinction is made here.